conda_packages/queryfuse/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: queryfuse
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '0.0.0') }}
+source:
+  # or from git:
+  git_url: https://github.com/yannpaul/QueryFuse.git
+  # once we have a release, uncomment next line and change it to match
+  # release tag name
+  git_tag: conda_work
```

Comment:
Note, after you accept this pull request, which is assentially a merge, you will need to edit line 6 to point to your git repository and comment out line 9 (until you add a version 1 tag in the git repository).